### ISSUE TITLE
Event logs does not work anymore

### ISSUE THREAD

**Issue by claustres** [kalisio/kEvent#37]
You are always asked to enter your interaction even when already done. It seems that multiple logs are generated in the DB instead of a single one.

This is probably linked to the migration to Feathers V3 and the new real-time event system based on channels. Something like a "ping-pong" effect might be involved.

**Comment by claustres** [comment]
It seems the real-time event is fired twice after logging the first interaction, causing the client to believe it has to jump to the next step again.

Initiated a diagram to describe the workflow: https://github.com/kalisio/kdk/blob/master/diagrams/Event%20Log%20Lifecycle.xml.

**Comment by claustres** [comment]
Upgrading feathers-reactive seems to have closed the issue, added a new safeguard test anyway: instead of just looking if the current step is different from the current state to initiate a log we now check if it is located after in the workflow.

**Comment by claustres** [comment]
It seems to appear randomly and be caused by a temporary state without any active log due to the fact the previous log is untagged before the new log is added: https://github.com/kalisio/kEvent/blob/master/src/hooks/hooks.event-logs.js#L30. Indeed, this trigger reactive updates and on the client side we now consider that if no active log is returned the workflow has to be initiated: https://github.com/kalisio/kEvent/blob/master/src/client/components/KEventCard.vue#L212.

We could either untag the previous log after inserting the new one or check on the client side if a log has been initialized before starting the workflow.

**Comment by claustres** [comment]
Untagging the previous log after inserting the new one still generates a reactive refresh without any active log although in the DB two logs still have the active flag.
